### ISSUE TITLE
"prefer" vs. "allow"

### ISSUE THREAD

**Issue by inexorabletash** [chrishtr/battery-savings#1]
Just a drive by from skimming the explainer:

The meta tag has: `allow-reduced-framerate`
The media query has: `prefer-reduced-framerate`

Should these align? So far as I can tell, they mean the same thing: if the UA is going to throttle the site, prefer to sacrifice frame rate before script speed or other techniques.

...

Which begs the question - should the naming be inverted? Should the site instead indicate what it prioritizes to keep rather than what to give up? e.g. "prefer-high-framerate", "prefer-script-speed", etc. I don't know the domain very well here, or how the set of options will grow over time and what knobs UAs actually have, so this may not make sense.

**Comment by chrishtr** [comment]
Good point about naming - I changed the explainer to suggest `reduced-framerate` for the media query, since the media query is for cases where for sure reduced framerate is in fact happening.

You raise good questions as well about which direction the preferences should go. I think we might end up needing preferences in both directions, as well as an `auto` mode when no preference was made.

**Comment by jimmywarting** [comment]
Read upon #3 of how to detect the mode

You have already introduced a css solution.

and a javascript solution to check if dark mode is enabled is throught matchMedia
```js
const {matches, onchange} = matchMedia('(prefers-color-scheme: dark)')
```

can only imagine a similar approach for prefer-reduced-framerate

```js
const {matches, onchange} = matchMedia('(prefer-reduced-framerate: true)')
```


But i think it's confusing to assume that prefer-reduced-framerate is the equivalent to battery saving
Would maybe want this instead
```js
const {matches, onchange} = matchMedia('(prefer-battery-saving: true)')
```

**Comment by jimmywarting** [comment]
we already have [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) and that is very similar to the term `prefer-reduced-framerate` imo

**Comment by CrazyPython** [comment]
When the OS goes into power saving mode, what does Chrome on Android currently do?
